code/game/gamemodes/cult/runes.dm

```diff
@@ -937,6 +937,7 @@ var/list/teleport_runes = list()
 	icon_state = "6"
 	construct_invoke = 0
 	color = rgb(200, 0, 0)
+	var/mob/living/summoned_guy = null
```

Comment:
why are you typecasting this as living, when it's only ever `/mob/living/carbon/human` it leads to stuff like below.